lib/applications/src/launcher.cpp

```diff
@@ -94,28 +99,28 @@ void applications::launcher::init() {
 }
 
 void applications::launcher::update() {
-    //std::cout << "launcher::update 1" << std::endl;
+    printf("start launcher update\n");
```

Comment:
convention std::cout pour serialcom
```suggestion
    std::cout << "start launcher update" << std::endl;
```